syntax/span.php

```diff
@@ -43,7 +43,7 @@ function render($mode, Doku_Renderer $renderer, $data) {
         if($mode == 'xhtml') {
             switch ($state){
                case DOKU_LEXER_ENTER:
-                if ($this->helper === null) {
+                if (!isset($this->helper)) {
```

Comment:
The helper property is not yet defined. Could you please add it to the class as well? If it is possible to define variable before use, that is preferable above checking before every use with isset().
 And probably the is null check will work again as well.
```
/** @var helper_plugin_folded */
protected $helper = null;
```

Reply:
Done.

Also I take back my comment about multiple use of the `helper` property. I misread the code and confused it with the `renderer` property.